mvpa2/algorithms/searchlight_hyperalignment.py

```diff
@@ -262,9 +263,10 @@ class SearchlightHyperalignment(ClassWithCollections):
             If None, hyperalignment is performed at every voxel (default).""")
 
     hyperalignment = Parameter(
-        Hyperalignment(ref_ds=0),
+        Hyperalignment(ref_ds=None),
         doc="""Hyperalignment instance to be used in each searchlight sphere.
             Default is just the Hyperalignment instance with default parameters.
+            Its `ref_ds` parameter would be overridden and therefore has no effect.
```

Comment:
If it passes tests, I have no problem merging it.

On Tue, Aug 23, 2016 at 6:52 PM, Feilong Ma <notifications@github.com>
wrote:

> In mvpa2/algorithms/searchlight_hyperalignment.py
> <https://github.com/PyMVPA/PyMVPA/pull/479#discussion_r75966452>:
>
> >          doc="""Hyperalignment instance to be used in each searchlight sphere.
> >              Default is just the Hyperalignment instance with default parameters.
> > +            Its `ref_ds` parameter would be overridden and therefore has no effect.
>
> Great idea!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/PyMVPA/PyMVPA/pull/479/files/6d67f88877304ea4c9031c043cd5a3f0c0eba6e7#r75966452>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAIfy70fdn0g_DAH3J5rvO4nkE0UL-Brks5qi3nFgaJpZM4I7Vn6>
> .
>